feature-topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/navigation/TopicNavigation.kt

```diff
@@ -24,20 +25,26 @@ import com.google.samples.apps.nowinandroid.core.navigation.NiaNavigationDestina
 import com.google.samples.apps.nowinandroid.feature.topic.TopicRoute
 
 object TopicDestination : NiaNavigationDestination {
-    override val route = "topic_route"
-    override val destination = "topic_destination"
     const val topicIdArg = "topicId"
+    override val route = "topic_route/{$topicIdArg}"
+    override val destination = "topic_destination"
+
+    fun createNavigationRoute(topicIdArg: String): String {
+        return "topic_route/$topicIdArg"
```

Comment:
Since we're consolidating these here, does it make sense to also add the `Uri.encode`/`Uri.decode` pair to handle ids with special characters?

Reply:
Sure thing! Added, thanks